inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/permute/permute_kernel_tile_8x8.cpp

```diff
@@ -42,6 +43,52 @@ ParamsKey PermuteKernel_tile_8x8::GetSupportedKey() const {
     return k;
 }
 
+std::vector<size_t> PermuteKernel_tile_8x8::GetBestLwsFromGws(const std::vector<size_t>& gws, const size_t tile_height, const size_t tile_width)
+{
+    std::vector<size_t> lws{1,1,1};
+
+    // float32 * tile_height * tile_width * 2 * work_items <= 64K
+    size_t max_num_work_items = 65536 / (4 * tile_height * tile_width * 2);
```

Comment:
I updated latest code to use one buf. So we dont need to use x2 any more. 
So max num work items will be min(256, (64*1024)/(sizeof(input type) * hight* width) )